spec/views/social_networking/profile_pages/show.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+module SocialNetworking
+  RSpec.describe "social_networking/profile_pages/show.html.erb", type: :view do
+    fixtures :all
```

Comment:
I didn't realize what I was stubbing.  For future reference, do you think it is a good rule of thumb to not mix fixtures and doubles or is that a case by case?  I can't think of any reason, if using `instance_doubles` that mixing would be bad.  In this case, it was unnecessary however.